zeppelin-web/src/app/notebook/notebook.html

```diff
@@ -61,6 +61,23 @@ <h5>Interpreter binding</h5>
                 </span>
               </small>
             </div>
+            <small ng-switch="item.status">
+              <small ng-switch-when="READY">
+                <i style="color: green; margin-right: 3px;" class="fa fa-circle"
```

Comment:
is `margin-right` required here ?

Reply:
@r-kamath  Thanks for taking time to reviewing the code, as per the margin-right concern it provides the breathing space between the interpreter list items and the (circle) which we show for interpreter status.